index.html

```diff
@@ -39,19 +39,42 @@
         <h2>Controls</h2>
         <h3>Date Range</h3>
         <form name="dateSelection">
-          <div class="dateRadio">
+          <div class="radioBtn">
             <input type="radio" name="radioDate" id="radioMonth" checked>
             <label for="radioMonth">Past 30 Days</label>
           </div>
-          <div class="dateRadio">
+          <div class="radioBtn">
             <input type="radio" name="radioDate" id="radioWeek">
             <label for="radioWeek">Past 7 Days</label>
           </div>
-          <div class="dateRadio">
+          <div class="radioBtn">
             <input type="radio" name="radioDate" id="radioDay">
             <label for="radioDay">Today</label>
           </div>
         </form>
+        <h3>Data Filter</h3>
+        <form name="typeSelection">
+          <div class="checkBtn">
```

Comment:
Ben is right, it would be awesome to generate this. For each key in `WPRDC_DATA_SOURCES` would generate these from the name of the key, the className, and the icon. Then, when assigning the `.type` to `record`, we could use the `dataSourceName` parameter in the `fetchWPRDCData` function